14-RemoteRepository/README.md

```diff
@@ -0,0 +1,145 @@
+# Working with remote repository
+
+## Introduction
+
+Once we have our remote repository configured we would be able to work with it.
+
+The list of commands below allows to work with remote repository.
+
+### Upstream
```

Comment:
I believe that most of the time in basic scenarios you would actually track the remote and have an upstream. I put there a comment that remote is optional...